run_multiqc_bp_qc.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -l
+
+#
+# A simple wrapper for the multiqc command in INS-22349
+# Usage: bash /proj/ngi2016001/private/scripts/run_multiqc_bp_qc.sh /proj/ngi2016001/nobackup/NGI/ANALYSIS/<project>
+#
+
```

Comment:
how about making this a sbatch script instead by adding the slurm parameters like this:

```suggestion
#SBATCH -A ngi2016001
#SBATCH -n 8
#SBATCH -t 05:00:00
#SBATCH -J run_multiqc_bp_qc
#SBATCH -o run_multiqc_bp_qc.%j.out
```